gfx.h

```diff
@@ -1402,6 +1402,24 @@ class GfxInternal
         device_->QueryInterface(IID_PPV_ARGS(&mesh_device_));
         SetDebugName(device_, "gfx_Device");
 
+        if((flags & kGfxCreateContextFlag_EnableDebugLayer) != 0)
+        {
+            ID3D12InfoQueue1 *debugCallback;
+            device_->QueryInterface(IID_PPV_ARGS(&debugCallback));
```

Comment:
Do we not need a `debugCallback->Release()` once we're done with this?

Reply:
yes, ill add that